scripts/build-visual-stories.js

```diff
@@ -13,8 +13,8 @@ const themes = [
     { value: 'sap_fiori_3', title: 'Quartz Light' },
     // { value: 'sap_fiori_3_dark', title: 'Quartz Dark' },
     // { value: 'sap_fiori_3_hcw', title: 'High Contrast White' },
-    { value: 'sap_fiori_3_hcb', title: 'High Contrast Black' }
-    //{ value: 'sap_horizon', title: 'Horizon' }
+    { value: 'sap_fiori_3_hcb', title: 'High Contrast Black' },
+    { value: 'sap_horizon', title: 'Morning Horizon' }
```

Comment:
We for sure will be overexciting limit very ofter with this new setting applied.

We should increase the amount of storyshots available per month or specify which stories will be built for certain pr, that's probably possible as we have `components` array in each storyshot file, that explicitly points to which components are used in the stories.

Reply:
FN components were removed from tests and one theme was added instead. I think this will more or less balance each other